src/service/xhr-impl.js

```diff
@@ -195,9 +176,11 @@ export class Xhr {
    *
    * @param {string} input
    * @param {?FetchInitDef=} opt_init
+   * @param {boolean=} opt_skipProcessing Disables any processing of the
+   *     response, including asserting the response is 2XX.
    * @return {!Promise<!JSONType>}
    */
-  fetchJson(input, opt_init) {
+  fetchJson(input, opt_init, opt_skipProcessing) {
```

Comment:
Yeah, I don't understand what this does. Is `fetchJson(input, init, true)` equivalent to `fetch(input, init, true)` in this world? If so, then surely we could just have the second one; this API change is only useful to clients who are asking for the raw response.

Or is the idea that `fetchJson(input, init, true)` should do the fetch and then transform the response body to JSON without regard to the status code? I think that would also be okay (although I'm uncertain whether anyone actually needs it, and having clients doing something different just use `fetch` seems fine). But I don't think that's what this does now.

Or am I missing something and the intended functionality is something else entirely?